### DISCUSSION TITLE
What is the hardware requirements for deploying a self hosting nextjs app?

### DISCUSSION THREAD

**Discussion by yunghzz35** [vercel/next.js#27265] [Help]
It seems there is not much information on hardware requirements for production. There is no guidelines on official documentation. I want to know the minimum cpu core and memory do I need to host nextjs.
From what I have observed when deploying on raspberry pi 4, when images are requested, nextjs consumes all 4 cores and running to 100%.

**Comment by dietrichmax** [comment]
I am also curious. It guess the Image component take some ressources because they images are optimized pretty slow on my website.

**Reply by wwwdepot** [reply]
Latest canary adds support for using sharp in image optimization https://github.com/vercel/next.js/releases/tag/v11.0.2-canary.20
